src/aws/auth.rs

```diff
@@ -9,6 +9,8 @@ use aws_config::{
     },
     sts::AssumeRoleProviderBuilder,
 };
+use aws_config::profile::profile_file::{ProfileFileKind, ProfileFiles};
+use aws_config::profile::ProfileFileCredentialsProvider;
```

Comment:
```suggestion
```

Duplicated from ln6-9

Reply:
Done. Not sure why CLion (or rustc) didn't yell at me about that